src/taxonomy/taxonomy-detail/TagListTable.jsx

```diff
@@ -0,0 +1,49 @@
+import {
+  DataTable,
+  TextFilter,
+} from '@edx/paragon';
+import Proptypes from 'prop-types';
+
+import { useTagListDataResponse, useTagListDataStatus } from '../api/hooks/selectors';
+
+const TagListTable = ({ taxonomyId }) => {
+  const useTagListData = () => {
+    const { isError, isFetched } = useTagListDataStatus(taxonomyId);
+    const tagList = useTagListDataResponse(taxonomyId);
+    return { isError, isFetched, tagList };
+  };
+
+  const { tagList } = useTagListData(taxonomyId);
+
+  if (!tagList || !tagList.results) {
+    return 'Loading...';
```

Comment:
I think you could use the [Paragon Spinner instead](https://paragon-openedx.netlify.app/components/spinner/)

Reply:
This code is outdated. I'm using the Loading status from the DataTable